apps/codeforafrica/src/lib/data/common/index.js

```diff
@@ -56,7 +58,10 @@ export async function getPageProps(api, context) {
   const settings = await api.findGlobal("settings");
   const navbar = getNavBar(settings);
   const footer = getFooter(settings);
-  const blocks = await blockify(page.blocks);
+  let blocks = await blockify(page.blocks);
+  if (params?.slugs?.length > 2) {
+    blocks = await pagify(slug, api, context);
+  }
```

Comment:
Been a while, so pagify also just returns blocks? I'd have though pagify creates a "stand-in" page that we can then process blocks from.